src/cart/utils/crt_utils.c

```diff
@@ -409,8 +411,8 @@ crtu_dc_mgmt_net_cfg_setenv(const char *name)
 	char			*ofi_interface;
 	char			*ofi_domain;
 	char			*cli_srx_set;
-	struct dc_mgmt_sys_info  crt_net_cfg_info;
-	Mgmt__GetAttachInfoResp *crt_net_cfg_resp;
+	struct dc_mgmt_sys_info  crt_net_cfg_info = {0};
```

Comment:
Intialization needed otherwise, if an error occur in dc_get_attach_info (line 415), the call to dc_put_attach_info (line 502) will segfault.